### PR TITLE
fix(dropdown): Set dropdown background color using css variable

### PR DESCRIPTION
**Pull request by jamcry**

### CODE REVIEW

**Review by mucahit — APPROVED** [review]
Thanks!

File: src/dropdown/_dropdown.scss [hunk 1]
```diff
@@ -43,7 +43,7 @@ $dropdown-header-horizontal-padding: 8px;
   width: 100%;
   height: var(--dropdown-header-button-height);
 
-  background-color: white;
+  background-color: var(--dropdown-header-button-bg);
   border: var(--dropdown-header-button-border-width) solid var(--default-border-color);
```
Comment (edizcelik) [inline src/dropdown/_dropdown.scss:47]: Should we have a variable for border-style as well ? 😄

File: src/dropdown/_dropdown.scss [hunk 2]
```diff
@@ -43,7 +43,7 @@ $dropdown-header-horizontal-padding: 8px;
   width: 100%;
   height: var(--dropdown-header-button-height);
```
Comment (edizcelik) [inline src/dropdown/_dropdown.scss:44]: I think we can define these via `--button-width` and `--button-height`, can we ?

Comment (mucahit) [reply src/dropdown/_dropdown.scss:44]: Agreed. Let's do it in another PR

File: src/dropdown/_dropdown.scss (hunk 1 above)
Comment (mucahit) [reply src/dropdown/_dropdown.scss:47]: Yeah maybe we can do something like this

```css
  --dropdown-header-button-border: var(--dropdown-header-button-border-width) solid var(--default-border-color)
```

```suggestion
  border: var(--dropdown-header-button-border);
```

**Review by bilgeyucel — APPROVED** [review]

**Pull request merged**